### ISSUE TITLE
[Game Crash] Crash occurs every time program is closed, 4.16

### ISSUE THREAD

**Issue by AngledLuffa** [TeamRizu/OutFox#613]
### Describe the crash

Every time I play the game, then close it, it crashes

### How can you reproduce the error?

1. do stuff
2. exit
3. crash

### Additional Information

crashinfo.txt   I'll attach more of the log if needed (but github doesn't allow posting the whole thing)

```
OutFox0.4-git-223b7a4a58 crash report (build 223b7a4a58, 20220801 @ 16:25:41)
--------------------------------------

Crash reason: A


debug resource file 'C': CreateFile failed (The system cannot find the file specified.).

Static log:
OutFox0.4-git-223b7a4a58
Compiled 20220801 @ 16:25:41 (build 223b7a4a58)
Log starting 2022-10-20 09:26:24
Loading window: sdl
Windows 10 - Running 64 Bit OutFox
OutFox Engine - Memory Detected: 32579MB Installed, (17392MB Available), 43843MB Swap Memory (23533MB Swap available)
Video driver: Intel(R) UHD Graphics 630 [Intel Corporation]
              26.20.100.7584, 11-26-2019 [PCI\VEN_8086&DEV_3E9B&SUBSYS_12741462]
Video driver: NVIDIA Quadro RTX 5000 with Max-Q Design [NVIDIA]
              30.0.14.7247, 10-26-2021 [pci\ven_10de&dev_1eb5&subsys_12741462]
Drive: "NVMe    SAMSUNG MZVLB1T0301Q" Driver: stornvme DMA: N/A
Sound device 0: Speakers (Realtek(R) Audio), 10.0, MID 1, PID 100 
WaveOut software mixing at 44100 hz
Sound driver: WaveOut
Lights driver: SystemMessage
Lights driver: Export
Video renderers: 'glad'
Graphics Manager:  OpenGL Vendor: NVIDIA Corporation
Graphics Manager:  OpenGL Renderer: Quadro RTX 5000 with Max-Q Design/PCIe/SSE2
Graphics Manager:  OpenGL Version: 3.1.0 NVIDIA 472.47
Graphics Manager:  OpenGL Max texture size: 32768
Graphics Manager:  OpenGL Texture units: 4
Graphics Manager:  OpenGL Extensions:
  GL_AMD: multi_draw_indirect, seamless_cubemap_per_texture, vertex_shader_layer, vertex_shader_viewport_index
  GL_ARB: ES2_compatibility, ES3_1_compatibility, ES3_2_compatibility, ES3_compatibility, arrays_of_arrays, 
    base_instance, bindless_texture, blend_func_extended, buffer_storage, cl_event, clear_buffer_object, 
    clear_texture, clip_control, color_buffer_float, compatibility, compressed_texture_pixel_storage, 
    compute_shader, compute_variable_group_size, conditional_render_inverted, conservative_depth, copy_buffer, 
    copy_image, cull_distance, debug_output, depth_buffer_float, depth_clamp, depth_texture, derivative_control, 
    direct_state_access, draw_buffers, draw_buffers_blend, draw_elements_base_vertex, draw_indirect, 
    draw_instanced, enhanced_layouts, explicit_attrib_location, explicit_uniform_location, 
    fragment_coord_conventions, fragment_layer_viewport, fragment_program, fragment_program_shadow, 
    fragment_shader, fragment_shader_interlock, framebuffer_no_attachments, framebuffer_object, framebuffer_sRGB, 
    geometry_shader4, get_program_binary, get_texture_sub_image, gl_spirv, gpu_shader5, gpu_shader_fp64, 
    gpu_shader_int64, half_float_pixel, half_float_vertex, imaging, indirect_parameters, instanced_arrays, 
    internalformat_query, internalformat_query2, invalidate_subdata, map_buffer_alignment, map_buffer_range, 
    multi_bind, multi_draw_indirect, multisample, multitexture, occlusion_query, occlusion_query2, 
    parallel_shader_compile, pipeline_statistics_query, pixel_buffer_object, point_parameters, point_sprite, 
    polygon_offset_clamp, post_depth_coverage, program_interface_query, provoking_vertex, query_buffer_object, 
    robust_buffer_access_behavior, robustness, sample_locations, sample_shading, sampler_objects, 
    seamless_cube_map, seamless_cubemap_per_texture, separate_shader_objects, shader_atomic_counter_ops, 
    shader_atomic_counters, shader_ballot, shader_bit_encoding, shader_clock, shader_draw_parameters, 
    shader_group_vote, shader_image_load_store, shader_image_size, shader_objects, shader_precision, 
    shader_storage_buffer_object, shader_subroutine, shader_texture_image_samples, shader_texture_lod, 
    shader_viewport_layer_array, shading_language_100, shading_language_420pack, shading_language_include, 
    shading_language_packing, shadow, sparse_buffer, sparse_texture, sparse_texture2, sparse_texture_clamp, 
    spirv_extensions, stencil_texturing, sync, tessellation_shader, texture_barrier, texture_border_clamp, 
    texture_buffer_object, texture_buffer_object_rgb32, texture_buffer_range, texture_compression, 
    texture_compression_bptc, texture_compression_rgtc, texture_cube_map, texture_cube_map_array, texture_env_add, 
    texture_env_combine, texture_env_crossbar, texture_env_dot3, texture_filter_anisotropic, 
    texture_filter_minmax, texture_float, texture_gather, texture_mirror_clamp_to_edge, texture_mirrored_repeat, 
    texture_multisample, texture_non_power_of_two, texture_query_levels, texture_query_lod, texture_rectangle, 
    texture_rg, texture_rgb10_a2ui, texture_stencil8, texture_storage, texture_storage_multisample, 
    texture_swizzle, texture_view, timer_query, transform_feedback2, transform_feedback3, 
    transform_feedback_instanced, transform_feedback_overflow_query, transpose_matrix, uniform_buffer_object, 
    vertex_array_bgra, vertex_array_object, vertex_attrib_64bit, vertex_attrib_binding, vertex_buffer_object, 
    vertex_program, vertex_shader, vertex_type_10f_11f_11f_rev, vertex_type_2_10_10_10_rev, viewport_array, 
    window_pos
  GL_ATI: draw_buffers, texture_float, texture_mirror_once
  GL_EXTX_framebuffer_mixed_formats
  GL_EXT: Cg_shader, abgr, bgra, bindable_uniform, blend_color, blend_equation_separate, blend_func_separate, 
    blend_minmax, blend_subtract, compiled_vertex_array, depth_bounds_test, direct_state_access, draw_buffers2, 
    draw_instanced, draw_range_elements, fog_coord, framebuffer_blit, framebuffer_multisample, 
    framebuffer_multisample_blit_scaled, framebuffer_object, framebuffer_sRGB, geometry_shader4, 
    gpu_program_parameters, gpu_shader4, import_sync_object, memory_object, memory_object_win32, 
    multi_draw_arrays, multiview_tessellation_geometry_shader, multiview_texture_multisample, 
    multiview_timer_query, packed_depth_stencil, packed_float, packed_pixels, pixel_buffer_object, 
    point_parameters, polygon_offset_clamp, post_depth_coverage, provoking_vertex, raster_multisample, 
    rescale_normal, secondary_color, semaphore, semaphore_win32, separate_shader_objects, separate_specular_color, 
    shader_image_load_formatted, shader_image_load_store, shader_integer_mix, shadow_funcs, sparse_texture2, 
    stencil_two_side, stencil_wrap, texture3D, texture_array, texture_buffer_object, texture_compression_dxt1, 
    texture_compression_latc, texture_compression_rgtc, texture_compression_s3tc, texture_cube_map, 
    texture_edge_clamp, texture_env_add, texture_env_combine, texture_env_dot3, texture_filter_anisotropic, 
    texture_filter_minmax, texture_integer, texture_lod, texture_lod_bias, texture_mirror_clamp, texture_object, 
    texture_sRGB, texture_sRGB_R8, texture_sRGB_decode, texture_shadow_lod, texture_shared_exponent, 
    texture_storage, texture_swizzle, timer_query, transform_feedback2, vertex_array, vertex_array_bgra, 
    vertex_attrib_64bit, win32_keyed_mutex, window_rectangles
  GL_IBM: rasterpos_clip, texture_mirrored_repeat
  GL_KHR: blend_equation_advanced, blend_equation_advanced_coherent, context_flush_control, debug, no_error, 
    parallel_shader_compile, robust_buffer_access_behavior, robustness, shader_subgroup
  GL_KTX_buffer_region
  GL_NVX: blend_equation_advanced_multi_draw_buffers, conditional_render, gpu_memory_info, gpu_multicast2, 
    linked_gpu_multicast, multigpu_info, nvenc_interop, progress_fence, shared_sync_object, sysmem_buffer
  GL_NV: ES1_1_compatibility, ES3_1_compatibility, alpha_to_coverage_dither_control, bindless_multi_draw_indirect, 
    bindless_multi_draw_indirect_count, bindless_texture, blend_equation_advanced, 
    blend_equation_advanced_coherent, blend_minmax_factor, blend_square, clip_space_w_scaling, command_list, 
    compute_program5, compute_shader_derivatives, conditional_render, conservative_raster, 
    conservative_raster_dilate, conservative_raster_pre_snap, conservative_raster_pre_snap_triangles, 
    conservative_raster_underestimation, copy_depth_to_color, copy_image, deep_texture3D, depth_buffer_float, 
    depth_clamp, draw_texture, draw_vulkan_image, explicit_multisample, feature_query, fence, fill_rectangle, 
    float_buffer, fog_distance, fragment_coverage_to_color, fragment_program, fragment_program2, 
    fragment_program_option, fragment_shader_barycentric, fragment_shader_interlock, framebuffer_mixed_samples, 
    framebuffer_multisample_coverage, geometry_shader4, geometry_shader_passthrough, gpu_multicast, gpu_program4, 
    gpu_program4_1, gpu_program5, gpu_program5_mem_extended, gpu_program_fp64, gpu_shader5, half_float, 
    internalformat_sample_query, light_max_exponent, memory_attachment, memory_object_sparse, mesh_shader, 
    multisample_coverage, multisample_filter_hint, occlusion_query, packed_depth_stencil, parameter_buffer_object, 
    parameter_buffer_object2, path_rendering, path_rendering_shared_edge, pixel_data_range, point_sprite, 
    primitive_restart, query_resource, query_resource_tag, register_combiners, register_combiners2, 
    representative_fragment_test, sample_locations, sample_mask_override_coverage, scissor_exclusive, 
    shader_atomic_counters, shader_atomic_float, shader_atomic_float64, shader_atomic_fp16_vector, 
    shader_atomic_int64, shader_buffer_load, shader_storage_buffer_object, shader_subgroup_partitioned, 
    shader_texture_footprint, shader_thread_group, shader_thread_shuffle, shading_rate_image, 
    stereo_view_rendering, texgen_reflection, texture_barrier, texture_compression_vtc, texture_env_combine4, 
    texture_multisample, texture_rectangle, texture_rectangle_compressed, texture_shader, texture_shader2, 
    texture_shader3, timeline_semaphore, transform_feedback, transform_feedback2, uniform_buffer_unified_memory, 
    vertex_array_range, vertex_array_range2, vertex_attrib_integer_64bit, vertex_buffer_unified_memory, 
    vertex_program, vertex_program1_1, vertex_program2, vertex_program2_option, vertex_program3, viewport_array2, 
    viewport_swizzle
  GL_OVR: multiview, multiview2
  GL_S3_s3tc
  GL_SGIS: generate_mipmap, texture_lod
  GL_SGIX: depth_texture, shadow
  GL_SUN_slice_accum
  GL_WIN_swap_hint
  WGL_EXT_swap_control
GLAD Windowed 1280x720 32 color 32 texture 60Hz Vsync NoSmoothLines
Outfox Engine: No Para Para Controller Found
Win32 Legacy - Found 0 XInput devices.
Win32 Legacy: Enumerating Direct Input device - Type: 0x00000413 Instance Name: "Keyboard" Product Name: "Keyboard"
Win32 Legacy - Found 1 DirectInput devices:
   0: 'Keyboard' axes: 0, hats: 0, buttons: 256 (buffered)
Disconnected: StepManiaX
StepManiaX
Auto-mapping all joysticks.
RageBitmapTexture: Couldn't load /AdditionalSongs/zjustdownloaded/Choopeta (Mamae eu quero)/Choopeta (Mamae eu quero).png: Unknown file format
Mixing 8000.717773 ahead in 12245 Mix() calls
Players joined: P1, P2
Language: en
Current renderer: OpenGL
Theme: default
Players joined: P1, P2
Language: en
Current renderer: OpenGL
Theme: default

Partial log:
04:47.934: TEXTUREMAN LEAK: '/Appearance/Themes/default/Graphics/LifeMeter frame.png', RefCount = 2.
04:47.934: TEXTUREMAN LEAK: '/Appearance/Themes/default/Graphics/LifeMeter markers.png', RefCount = 2.
04:47.934: TEXTUREMAN LEAK: '/Appearance/Themes/default/Graphics/LifeMeterBar under.png', RefCount = 2.
04:47.934: TEXTUREMAN LEAK: '/Appearance/Themes/default/Graphics/StreamDisplay hot.png', RefCount = 2.
04:47.934: TEXTUREMAN LEAK: '/Appearance/Themes/default/Graphics/StreamDisplay normal.png', RefCount = 4.
04:47.996: FoxClock: Game uptime was 4 minutes, 47 seconds
04:47.996: Players joined: P1, P2
04:47.996: Language: en
04:47.996: Current renderer: OpenGL
04:47.996: Theme: default

-- End of report
```

**Comment by Scraticus** [comment]
Fixed in 4.18, many thanks for reporting

**Comment by AngledLuffa** [comment]
Excellent, ty!  Looking forward to it.
